learners/BF_measles.Rmd

```diff
@@ -0,0 +1,884 @@
+    disease, leading to a faster spread within the community.
+
+4.  The $\mathcal{R}_0$ value helps to determine the finalsize- the
+    total number of individuals that that will be infected- of the
+    outbreak via the following equation [$$
+    \begin{equation}
+    z = 1 - e^{-\mathcal{R}_0 \cdot z}
+    \end{equation}
+     \qquad(3)$$]{#eq-fs}
+
+Where:
+
+-   $z$ is the final size (the fraction of the population infected),
+
+-   $\mathcal{R}_0$ is the basic reproduction number, and
+
+-   $e$ is the base of the natural logarithm.
+
+5.  Additionally, the basic reproduction number $\mathcal{R}_0$ is
+    crucial for determining the herd immunity threshold (HIT), which
+    represents the minimum proportion of the population that must be
+    immune (either through vaccination or previous infection) in order
+    to contain the epidemic and prevent widespread transmission. The
+    formula for the herd immunity threshold is:
+    [$$ HIT = 1 - \frac{1}{\mathcal{R}_0} \qquad(4)$$]{#eq-hit}
+
+Where:
+
+-   HIT is the proportion of the population that needs to be immune to
+    achieve herd immunity,
+
+As $\mathcal{R}_0$ increases, the herd immunity threshold also
+increases, meaning a larger proportion of the population must be immune
+to prevent the epidemic from spreading.
+
+# Measles Outbreak in Burkina Faso
+
+## Background
+
+Measles is a highly contagious viral disease that spreads quickly,
+particularly in environments where people live or gather in close
+quarters, such as schools, healthcare facilities, or densely populated
+urban areas. Transmission occurs through respiratory droplets when an
+infected person coughs or sneezes, and the virus can remain viable in
+the air for up to two hours. Effective vaccines are available, offering
+an efficacy of 93% to 97%, making vaccination a key tool in controlling
+measles outbreaks.
+
+Given the virus's high transmissibility, even a single case of measles
+can trigger large outbreaks, especially in regions with low vaccination
+coverage. The latent (pre-infectious) period for measles is
+typically around 8 days, while the infectious period averages 5 days ([Masters et al., 2023](https://www.thelancet.com/journals/lanpub/article/PIIS2468-2667(23)00130-5/fulltext)). The SEIR model in
+[Equation 1](#eq-s1) is commonly employed to simulate the
+dynamics of measles outbreaks and evaluate control measures like
+vaccination
+
+## Burkina Faso Situation in Jan 2024
+
+Burkina Faso, like many other countries in Africa, faces recurring
+measles outbreaks. This can be attributed to factors such as limited
+healthcare access, inadequate vaccination coverage, and fragile
+healthcare systems. In 2023, Burkina Faso experienced a particularly
+severe outbreak, with over 2,000 suspected cases reported ([Crisis24, 2024](https://crisis24.garda.com/alerts/2024/02/burkina-faso-elevated-measles-activity-reported-in-multiple-areas-in-february)),
```

Comment:
Throughout the example, we focus on infections - worth making it clear how this might related to reported cases?

Reply:
.... In these simulations, we will focus specifically on the number of infections, which represents a subset of the total reported suspected cases.